src/view/gui/ui_fPopulationAtlasDialog.h

```diff
@@ -202,8 +155,7 @@ class ui_fPopulationAtlasDialog
    // inputAtlasLabel->setText(QApplication::translate("fPopulationAtlasDialog", "Atlas file:", 0, QApplication::UnicodeUTF8));
     // NEW CHANGES
     outputDirectoryLabel->setText(QApplication::translate("fPopulationAtlasDialog", "Output Directory:", 0));
-    inputDirectoryLabel->setText(QApplication::translate("fPopulationAtlasDialog", "Input Directory:", 0));
-    inputlabelLabel->setText(QApplication::translate("fPopulationAtlasDialog", "Label File:", 0));
+    inputlabelLabel->setText(QApplication::translate("fPopulationAtlasDialog", "Input File:", 0));
```

Comment:
Please change the 'inputlabellabel' name to something that is easy to understand. Perhaps to inputfilelabel?

Reply:
replaced with a meaningful label 'inputfileLabel'